tests/v2/validation/provisioning/provisioning_node_driver_test.go

```diff
@@ -33,13 +34,14 @@ type RKE2NodeDriverProvisioningTestSuite struct {
 }
 
 const (
-	baseDOClusterName       = "testautomationdocluster"
-	baseAWSClusterName      = "testautomationawscluster"
-	namespace               = "fleet-default"
-	defaultRandStringLength = 5
+	baseDOClusterName        = "testautomationdocluster"
+	baseAWSClusterName       = "testautomationawscluster"
+	baseHarvesterClusterName = "automationharvester-"
```

Comment:
Let's have these all consistent, we can name them all `automation(provider)-` but just keep them consistent